velox/functions/prestosql/tests/ArrayAverageTest.cpp

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <optional>
+#include "velox/functions/prestosql/tests/utils/FunctionBaseTest.h"
+
+using namespace facebook::velox;
+using namespace facebook::velox::test;
+using namespace facebook::velox::functions::test;
+
+namespace {
+
+class ArrayAverageTest : public FunctionBaseTest {
+ protected:
+  // Evaluate an expression.
+  template <typename T>
+  void testExpr(
+      const VectorPtr& expected,
+      const std::string& expression,
+      const std::vector<VectorPtr>& input) {
+    auto result = evaluate<FlatVector<T>>(expression, makeRowVector(input));
+    assertEqualVectors(expected, result);
+  }
+};
+
+} // namespace
+
+// Test floating point arrays
```

Comment:
Comments should end in a period. Would you fix throughout?

Reply:
Oh, my apologies. I missed this part, will update all of the comments.